lib/engines/content_block_manager/app/views/content_block_manager/content_block/editions/workflow/review.html.erb

```diff
@@ -20,14 +20,21 @@
   <div class="govuk-grid-column-full">
     <%= form_with(url: content_block_manager.content_block_manager_content_block_workflow_path(@content_block_edition, step: ContentBlockManager::ContentBlock::Editions::WorkflowController::NEW_BLOCK_STEPS[:review]), method: :put) do %>
       <div class="govuk-button-group govuk-!-margin-bottom-6">
-        <%= render "govuk_publishing_components/components/button", {
-          text: "Accept and publish",
-          name: "accept_and_publish",
-          value: "Accept and publish",
-          type: "submit",
-        } %>
-        <%= link_to("Cancel", content_block_manager.content_block_manager_content_block_documents_path, class: "govuk-link") %>
+        <div>
```

Comment:
Do we need these `<div>`s here?

Reply:
I think so for the `govuk-button-group` to align the items correctly, because I've had to put that `<% end %>` of the form in before the Cancel button - otherwise the second button goes wonky.

Reply:
Aaah, makes sense